### PR TITLE
Application V1 Files added

### PR DESCRIPTION
**Pull request by ataur-rk25**

### CODE REVIEW

File: users/assets/js/script.js [hunk 1]
```diff
@@ -0,0 +1,303 @@
+document.addEventListener("DOMContentLoaded", function() {
+
+    var passwordInput = document.getElementById("password");
```
Comment (burhanuddin7) [inline users/assets/js/script.js]: Avoid multiple var variable declaration lines, declare multiple variable in one line separated by a comma. Please check all multiple variables declaration in this file.

File: users/assets/js/script.js [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+        });
+    }
+    if(document.body.contains(confirmInput)){
+        confirmInput.addEventListener("keyup", function() {
+            confirmPassword(passwordInput.value, confirmInput.value);
+        });
+    }
+
+     //Email
+    if(document.body.contains(emailInput)){
```
Comment (burhanuddin7) [inline users/assets/js/script.js]: Prettify All PHP and JS files.

File: users/assets/js/script.js [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+                success: function(response) {
+                    console.log(response);
+                    jQuery('#action_message').text(response);
+                    window.location.href = 'user_list.php';
+                    loadUserTable();
+                }
+            });
+        });
+
+        jQuery(document).on("click", "#edit,#add", function() {
```
Comment (burhanuddin7) [inline users/assets/js/script.js]: Avoid using third party JQuery library, use pure vanilla javascript

File: users/assets/js/script.js [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+
+function validateForm() {
+    var username = document.getElementById("username").value;
+    var name = document.getElementById("name").value;
+    var email = document.getElementById("email").value;
+    var password = document.getElementById("password").value;
+    var confirm_password = document.getElementById("confirm_password").value;
+
+    if (username === "" || name === "" || email === "") {        
+        document.querySelectorAll("#username_error,#name_error,#email_error").forEach(function(element) {
```
Comment (burhanuddin7) [inline users/assets/js/script.js]: use a common class for error message instead of using individual ID's

File: users/assets/css/style.css [hunk 5]
```diff
@@ -0,0 +1,261 @@
...
+    font-size: 16px;
+}
+h1{
+    font-size: 48px;
+}
+h2{
+    font-size: 40px;
+}
+h3{
+    font-size: 32ßpx;
```
Comment (burhanuddin7) [inline users/assets/css/style.css]: please check the value here of the css property

File: users/login.php [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+                    echo '<p style="color: red;">' . $_SESSION['error_message'] . '</p>';
+                    unset($_SESSION['error_message']); // Clear the error message after displaying
+                }
+            ?>
+            <form action="login_process.php" method="post">
+                <div class="form-group-row">        
+                    <div class="form-group">
+                        <input type="text" name="username" id="username" placeholder="Username"> 
+                        <p class="text-center">OR</p>
+                        <input type="email" name="email" id="email" placeholder="Email" pattern="/^(?=.*[A-Z])(?=.*\d)(?=.*[!@#$%()]).{6,}$/">
```
Comment (burhanuddin7) [inline users/login.php]: can we add individual error messages to the fields respectively and clear the error on key up, also please higight the textbox with error border

File: users/assets/js/script.js [hunk 7]
```diff
@@ -0,0 +1,303 @@
...
+
+        jQuery(document).on("click", "#edit_my_details", function() {
+            jQuery('#edit_user_form').toggle();
+            var editFormOffset = jQuery("#edit_user_form").offset().top;
+            jQuery("html, body").animate({ scrollTop: editFormOffset }, 1000);
+        });
+
+
+        function loadUserTable() {
+            $.ajax({
```
Comment (burhanuddin7) [inline users/assets/js/script.js]: can we hit this function only once when every other field has been validated in the front end, till then we can avid multiple AJAX calls by disabling the submit button

**Review by burhanuddin7 — CHANGES_REQUESTED** [review]

File: users/assets/js/script.js [hunk 8]
```diff
@@ -0,0 +1,303 @@
...
+    if (!emailValid.test(email)) {
+        document.querySelector("#email_error").innerText = "Email ID is not valid";
+        return false;
+    } else {
+        document.querySelector("#email_error").innerText = "";
+    }
+    return true;
+}
+
+function removeChars(phoneInput){
```
Comment (nkanvadiya) [inline users/assets/js/script.js]: Use prettify all JS and PHP file

File: users/assets/js/script.js [hunk 9]
```diff
@@ -0,0 +1,303 @@
...
+        });
+
+
+        function loadUserTable() {
+            $.ajax({
+                url: "get_users.php",
+                method: "GET",
+                dataType: "json",
+                success: function(data) {
+                    console.log(data);
```
Comment (nkanvadiya) [inline users/assets/js/script.js]: remove all console or debug log disable for review

File: users/user_list.php [hunk 10]
```diff
@@ -0,0 +1,44 @@
+<?php   
+session_start();  
+if(!isset($_SESSION["user_id"])){  
+    header("location:login.php");  
+} else {  
+    if(isset($_SESSION['user_role']) && $_SESSION['user_role']=='admin'){
+?>
+<?php
```
Comment (nkanvadiya) [inline users/user_list.php:1]: No need PHP End and Open here

File: users/assets/js/script.js (hunk 1 above)
Comment (ataur-rk25) [reply users/assets/js/script.js]: This is fixed

File: users/assets/js/script.js (hunk 2 above)
Comment (ataur-rk25) [reply users/assets/js/script.js]: Done

File: users/assets/js/script.js (hunk 3 above)
Comment (ataur-rk25) [reply users/assets/js/script.js]: Removed

File: users/assets/js/script.js (hunk 4 above)
Comment (ataur-rk25) [reply users/assets/js/script.js]: This is fixed

File: users/assets/css/style.css (hunk 5 above)
Comment (ataur-rk25) [reply users/assets/css/style.css]: This is fixed

File: users/login.php (hunk 6 above)
Comment (ataur-rk25) [reply users/login.php]: This is done

File: users/assets/js/script.js (hunk 7 above)
Comment (ataur-rk25) [reply users/assets/js/script.js]: This is fixed

File: users/assets/js/script.js (hunk 8 above)
Comment (ataur-rk25) [reply users/assets/js/script.js]: This is done

File: users/assets/js/script.js (hunk 9 above)
Comment (ataur-rk25) [reply users/assets/js/script.js]: This is fixed

File: users/user_list.php (hunk 10 above)
Comment (ataur-rk25) [reply users/user_list.php:1]: Done

Comment (nkanvadiya) [reply users/user_list.php:1]: LGTM

File: users/assets/js/script.js (hunk 9 above)
Comment (nkanvadiya) [reply users/assets/js/script.js]: LGTM

File: users/assets/js/script.js (hunk 8 above)
Comment (nkanvadiya) [reply users/assets/js/script.js]: LGTM

**Pull request merged**